src/native/windows/apps/prunsrv/prunsrv.c

```diff
@@ -2204,8 +2217,11 @@ void __cdecl main(int argc, char **argv)
     gStdwrap.szLogPath = SO_LOGPATH;
     /* Only redirect when running as a service */
     if (lpCmdline->dwCmdIndex == 2) {
+        apxLogWrite(APXLOG_MARK_DEBUG "redirect!");
```

Comment:
I don't think we need exclamations in debug log messages! 😉 

More seriously, I think this kind of log message is unhelpful. How about saying _what_ we are redirecting to _where_?